src/sync.py

```diff
@@ -54,8 +62,10 @@ def sync_from_specified_source(source: 'Issue', destination: 'Issue'):
     def sync_from_most_current(a: 'Issue', b: 'Issue'):
 
         if a.updated > b.updated:  # a is the most current
+            print(f'syncing {b} (updated at {b.updated}) from {a} (updated at {a.updated})')
```

Comment:
Can you change this to be logged rather than printed. Same right below.

Reply:
Sure. I will make a ticket for having logging throughout